### PR TITLE
tuya MIR-HE200-TY - Changed motion from enum to binary for compatibility in Symcon

### PR DESCRIPTION
**Pull request by Burki24**

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -1658,7 +1658,7 @@ module.exports = [
         toZigbee: [tz.tuya_radar_sensor],
         exposes: [
             e.illuminance_lux(), e.presence(),
-            exposes.enum('motion', ea.STATE, ['true', 'false']).withDescription('moving inside the range of the sensor'),
+            exposes.binary('motion', ea.STATE, ['true', 'false']).withDescription('moving inside the range of the sensor'),
```
Comment (Koenkk) [inline devices/tuya.js:1661]: Please update the converter to expose the bool as a bool and not a string and change this to `exposes.binary('motion', ea.STATE, [true, false])`

Comment (Burki24) [reply devices/tuya.js:1661]: Hi Koenkk,

thanks for your help. The converter allready expose it as bool. So I've made only the mistake inside /devices/tuya.js.

Comment (Koenkk) [reply devices/tuya.js]: I think the value is a string not a bool, please update https://github.com/Koenkk/zigbee-herdsman-converters/blob/d02a164ae94a0c51adca8ea31ff7aae141abe637/converters/fromZigbee.js#L7314 and to the same for `presence`

**Comment by Burki24** [comment]
Hopefully, I finished that.

**Comment by Koenkk** [comment]
Thanks!

**Comment by Burki24** [comment]
Thanks back for bringing me to the right way.

**Pull request merged**